Sources/TuistKit/ProjectEditor/ProjectEditorMapper.swift

```diff
@@ -79,13 +81,20 @@ final class ProjectEditorMapper: ProjectEditorMapping {
                                                      targetSettings: targetSettings,
                                                      sourcePaths: [configPath])
         }
+        var dependenciesTarget: Target?
+        if let dependenciesPath = dependenciesPath {
+            dependenciesTarget = Target.editorHelperTarget(name: "Dependencies",
```

Comment:
What about making `editorHelperTarget` a private method in `ProjectEditorMapper`. Considering that method is scoped to this file, I think it's more appropiate to define it here.